cfpb/consumerfinance.gov
# Mortgage Performance Trends: Ignore charts that have already been initialized

Mortgage Performance Trends calls `createChart` from cfpb-chart-builder, but cfpb-chart-builder calls `createCharts` on page load, so charts with the class `cfpb-chart` have the possibility of being initialized twice.

See this error in production by visiting https://www.consumerfinance.gov/data-research/mortgage-performance-trends/mortgages-30-89-days-delinquent/ and reloading the page and checking the dev console.

## Changes

- Mortgage Performance Trends: Ignore charts that have already been initialized.


## How to test this PR

1. `yarn build` and visit http://localhost:8000/data-research/mortgage-performance-trends/mortgages-30-89-days-delinquent/ and check that the two charts work and that there are no console errors.
